source/playgrounds.txt

```diff
@@ -150,19 +150,48 @@ If you have a playground file open in |vscode-short| and do not have
 an active connection, |vsce| displays
 :guilabel:`Click here to add connection` at the top of your playground.
 
-.. figure:: /images/vsce-playground-add-connection-btn.png
-   :figwidth: 700px
-   :alt: Link to add connection from playground
-
-1. Click this link to open the connection string dialog.
-
-#. Enter the
-   :manual:`connection string </reference/connection-string/>` for the
-   deployment you want to run this playground against.
+.. procedure::
+   :style: normal
    
-#. Press :guilabel:`Enter`.
-
-#. Run your playground.
+   .. step:: Select :guilabel:`Click here to add connection` 
+
+      After you click :guilabel:`Click here to add connection`, |vsce| displays 
+      the connection string drop-down menu.
+
+      .. figure:: /images/vsce-playground-add-connection-btn.png
+         :figwidth: 700px
+         :alt: Link to add connection from playground
+
+   .. step:: Click :guilabel:`Add new connection`
+
+      In the connection string drop-down menu, select 
+      :guilabel:`Add new connection` to enter a new connection string. If you 
+      have previously connected to the playground with another deployment, the 
+      connection string will appear as an option under the drop-down menu. 
+
+   .. step:: Enter your connection string
+      
+      Enter the :manual:`connection string </reference/connection-string/>` to 
+      connect to your deployment.
+
+      .. note:: 
+         
+         If the connection string specifies a database the playground runs 
+         against that database by default. To switch databases,   
+         call ``use('<database_name>')``.
+
+         If you are connected to a default database, the playground 
+         autocompletes only for collection names available on that 
+         database. 
+         
+   .. step:: Press :guilabel:`Enter`
+
+      After you enter your connection string, a confirmation message replaces 
```

Comment:
Nit, fine as is, in VSCode these `confirmation` messages are called `code lenses`. https://code.visualstudio.com/blogs/2017/02/12/code-lens-roundup

Reply:
Gotcha, thanks! Will update.